templates/types/streaming/fastapi/app/api/routers/models.py

```diff
@@ -25,7 +25,7 @@ class File(BaseModel):
     filetype: str
 
 
-class AnnotationData(BaseModel):
+class AnnotationFileData(BaseModel):
```

Comment:
why this change?

Reply:
This data specified for file data type for now so i think better to call it as AnnotationFileData (no-breaking change here).
So generally: `AnnotationData = AnnotationFileData | List[str]`
Where List[str] is specified for list of suggested questions for now, we can also explicitly create another type for the suggested question.